library/src/main/java/com/getstream/sdk/chat/ChatSocketListener.kt

```diff
@@ -141,7 +141,6 @@ internal class ChatSocketListener(
             is ConnectingEvent,
             is DisconnectedEvent,
             is ErrorEvent,
-            is ChannelUpdatedByUserEvent,
```

Comment:
This was a duplicate type in this branch of the `when` statement, just cleaned up a warning by removing it